Why the compact job failed in our test?

Was it expected that the  would fail? Should we fix the test case?

**Describe the bug (__must be provided__)**
```
platform linux -- Python 3.6.8, pytest-5.3.2, py-1.9.0, pluggy-0.13.1 -- /usr/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.6.8', 'Platform': 'Linux-5.4.109-1.el7.elrepo.x86_64-x86_64-with-centos-7.9.2009-Core', 'Packages': {'pytest': '5.3.2', 'py': '1.9.0', 'pluggy': '0.13.1'}, 'Plugins': {'metadata': '1.8.0', 'html': '2.0.1', 'reportlog': '0.1.0', 'forked': '1.3.0', 'xdist': '1.31.0', 'benchmark': '3.2.3', 'drop-dup-tests': '0.3.0', 'bdd': '4.0.2', 'yapf3': '0.5.1'}, 'CI': 'true'}
benchmark: 3.2.3 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /__w/nebula/nebula/tests, inifile: pytest.ini
plugins: metadata-1.8.0, html-2.0.1, reportlog-0.1.0, forked-1.3.0, xdist-1.31.0, benchmark-3.2.3, drop-dup-tests-0.3.0, bdd-4.0.2, yapf3-0.5.1
collecting ... collected 3 items

tck/steps/test_jobs.py::test_submit_job_require_space PASSED             [ 33%]
tck/steps/test_jobs.py::test_not_existed_job PASSED                      [ 66%]
tck/steps/test_jobs.py::test_submit_and_show_jobs FAILED                 [100%]

=================================== FAILURES ===================================
__________________________ test_submit_and_show_jobs ___________________________
/github/home/.local/lib/python3.6/site-packages/pytest_bdd/scenario.py:177: in scenario_wrapper
    _execute_scenario(feature, scenario, request, encoding)
/github/home/.local/lib/python3.6/site-packages/pytest_bdd/scenario.py:143: in _execute_scenario
    _execute_step_function(request, scenario, step, step_func)
/github/home/.local/lib/python3.6/site-packages/pytest_bdd/scenario.py:113: in _execute_step_function
    return_value = step_func(**kwargs)
tck/conftest.py:492: in result_should_be_in_order_and_register_key
    result_ds = cmp_dataset(request, graph_spaces, result, order=True, strict=True, contains=CmpType.CONTAINS, first_n_records=n)
tck/conftest.py:341: in cmp_dataset
    assert res, "\n".join(msg)
E   AssertionError: Fail to exec: SHOW JOBS;
E     Response: |Job Id|Command|Status|Start Time|Stop Time|
E     0: |34|STATS|FINISHED|2021-09-16T07:14:31.000000|2021-09-16T07:14:31.000000|
E     1: |33|FLUSH|FINISHED|2021-09-16T07:14:30.000000|2021-09-16T07:14:30.000000|
E     2: |32|COMPACT|FAILED|2021-09-16T07:14:29.000000|2021-09-16T07:14:29.000000|
E     Expected: |Job Id|Command|Status|Start Time|Stop Time|
E     0: |re.compile('\\d+')|STATS|FINISHED|re.compile('\\w+')|re.compile('\\w+')|
E     1: |re.compile('\\d+')|FLUSH|FINISHED|re.compile('\\w+')|re.compile('\\w+')|
E     2: |re.compile('\\d+')|COMPACT|FINISHED|re.compile('\\w+')|re.compile('\\w+')|
E     NotFoundRow: 
E     Location: job/Job.feature:60
E     Space: {'name': 'EmptyGraph_X3DYTI', 'vid_type': 'FIXED_STRING(20)', 'partition_num': 9, 'replica_factor': 1, 'charset': 'utf8', 'collate': 'utf8_bin'}
E     vid_fn: None
E   assert False
```

**Your Environments (__must be provided__)**

ci/centos7

--- a/tests/tck/job/Job.feature
+++ b/tests/tck/job/Job.feature
@@ -50,6 +50,7 @@ Feature: Submit job space requirements
       | partition_num  | 9                |
       | replica_factor | 1                |
       | vid_type       | FIXED_STRING(20) |
+    And wait 6 seconds
     When executing query:
       """
       SUBMIT JOB COMPACT;

